ci/licenses_golden/licenses_flutter

```diff
@@ -1370,12 +1370,8 @@ FILE: ../../../flutter/shell/platform/fuchsia/flutter/runner_tzdata_unittest.cc
 FILE: ../../../flutter/shell/platform/fuchsia/flutter/runner_unittest.cc
 FILE: ../../../flutter/shell/platform/fuchsia/flutter/surface.cc
 FILE: ../../../flutter/shell/platform/fuchsia/flutter/surface.h
-FILE: ../../../flutter/shell/platform/fuchsia/flutter/task_observers.cc
-FILE: ../../../flutter/shell/platform/fuchsia/flutter/task_observers.h
 FILE: ../../../flutter/shell/platform/fuchsia/flutter/task_runner_adapter.cc
```

Comment:
You should be able to delete "task_runner_adapter" too.  Nothing (except for platform_view_unittest) uses them anymore after your change, and the test can just use an fml::MessageLoop now.

Reply:
Woops, I just re-read the PR description